src/components/overlay/pages/overlay-interactive.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import DocsActiveBlock from 'components/DocsActiveBlock';
+import Overlay from '../Overlay';
+
+const Overlays = () => {
+  const settings = [];
+
+  return <div>
+    <DocsActiveBlock settings={settings}>
+      <div>
+        <Overlay/>
```

Comment:
huh, that's how we use overlay? I'd make more sense to make it wrap content it's covering IMO. I'd make it more flexible (you can cover only part of page). @anna-wro @matikrk @aju WDYT?